etl-job/fhir_import_export.py

```diff
@@ -215,89 +212,56 @@ def _load_all(study: str,
         file_path = str(file_path)
         extraction_path = str(extraction_path)
         output['logs'].append(f"Simplifying study: {file_path}")
-        simplify_directory(file_path, pattern="**/*.*",
-                           output_path=extraction_path,
-                           schema_path=schema, dialect='PFB',
-                           config_path='config.yaml')  # Don't want to add this Iceberg pr right now split_obs=False
 
-        meta_upload(source_path=extraction_path,
-                    program=program, project=project,
-                    silent=False, dictionary_path=schema, config_path=config)
+        subprocess.run(["jsonschemagraph", "gen-dir", "iceberg/schemas/graph", f"{file_path}", f"{extraction_path}","--project_id", f"{project_id}","--gzip_files"])
+        bulk_load(_get_grip_service(), "CALIPER",f"{program}-{project}", extraction_path, output, _get_token())
 
         assert pathlib.Path(work_path).exists(), f"Directory {work_path} does not exist."
         work_path = pathlib.Path(work_path)
         db_path = (work_path / "local_fhir.db")
         db_path.unlink(missing_ok=True)
 
         db = LocalFHIRDatabase(db_name=db_path)
-
-        db.load_ndjson_from_dir(path=file_path)
+        db.bulk_insert_data(resources=get_project_data(_get_grip_service(), "CALIPER", f"{program}-{project}", output, _get_token()))
 
         index_generator_dict = {
             'researchsubject': db.flattened_research_subjects,
             'specimen': db.flattened_specimens,
             'file': db.flattened_document_references
         }
 
+        # To ensure differences in the dataframer versions do not conflict, clear the project, and reload the project.
+        for index in index_generator_dict.keys():
+            meta_flat_delete(project_id=f"{program}-{project}", index=index)
```

Comment:
One thing to note: if this FHIR job fails midway, then you're only left with the indices that were able to be loaded. Is this the expected behavior that we want for upload?

Also, does this have performance detriments / is index generation relatively fast?

Reply:
Not sure. Maybe create an issue and discuss with @bwalsh and others?

Reply:
okay will write as a separate issue since an incomplete / failed job in previous versions still results in a borked ES index. See #40